spec/gtp_writer_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'byebug'
+
+RSpec.describe SGF::GtpWriter do
+  let(:parser) { SGF::Parser.new }
+  let(:suffix) { '' }
+
+  TEMP_FILE = 'spec/data/temp.gtp'
+
+  after { FileUtils.rm_f TEMP_FILE }
```

Comment:
Instead of doing this you could just use https://ruby-doc.org/stdlib-1.9.3/libdoc/tempfile/rdoc/Tempfile.html .

Reply:
its a good suggestion, but I'm reusing the same pattern you have in the `spec/writer_spec.rb`

Reply:
Heh. Fair enough.